src/containers/bitset.c

```diff
@@ -534,7 +534,7 @@ uint16_t bitset_container_minimum(const bitset_container_t *container) {
 }
 
 /* Returns the largest value (assumes not empty) */
-inline uint16_t bitset_container_maximum(const bitset_container_t *container) {
```

Comment:
This inline in a .c file chokes Visual Studio. I don't think it's doing anything in other C/C++ compilers since the implementation is not in the .h file so I removed it. Another fix would be to add the keyword and the implementation to the .h file.